db/schema/patch-007-008.sql

```diff
@@ -0,0 +1,241 @@
+-- Add a lockedAt column for account locking.
+
+ALTER TABLE accounts ADD COLUMN lockedAt BIGINT UNSIGNED DEFAULT NULL;
+
+-- the codes to unlock an account
+CREATE TABLE IF NOT EXISTS accountUnlockCodes (
```

Comment:
Even though we have `IF NOT EXISTS` in our earlier patches when creating tables, I'm trying to move us away from any SQL statements in our patches that are non-deterministic. ie. if this patch runs, each and every statement should do the thing it is intending to do (and we're never in the situation where we say "did it or didn't it do anything?" Best if that's removed.